charts/spire/charts/spire-server/values.yaml

```diff
@@ -684,6 +684,25 @@ nodeAttestor:
   joinToken:
     ## @param nodeAttestor.joinToken.enabled Enable the join_token nodeattestor
     enabled: false
+  tpmDirect:
+    ## @param nodeAttestor.tpmDirect.enabled Enable the direct tpm nodeattestor
+    enabled: false
+    ## @param nodeAttestor.tpmDirect.image.registry The OCI registry to pull the image from
+    ## @param nodeAttestor.tpmDirect.image.repository The repository within the registry
+    ## @param nodeAttestor.tpmDirect.image.pullPolicy The image pull policy
+    ## @param nodeAttestor.tpmDirect.image.tag Overrides the image tag
+    ##
+    image:
+      registry: docker.io
+      repository: boxboat/spire-tpm-plugin-tpm-attestor-server
+      pullPolicy: IfNotPresent
+      tag: "v1.7.0"
+    ## @param nodeAttestor.tpmDirect.checksum The sha256 checksum of the plugin binary
+    checksum: 6dc74979eb41ecbd45cd047592c24e1830f9c52fff40441d69870e8927def13f
```

Comment:
Is this something that needs to be calculated by the user?

Reply:
SPIRE's config file needs a checksum to verify the plugin has not been tempered with. Since SPIRE itself requires the user to set it, I'm following that convention here. The user can calculate it, but its a little tedious to do so. We set the correct checksum for the image we're recommending here so the user doesn't have to. Ideally the checksum would be provided by the provider of the image. I'll have a chat with them.

Reply:
https://github.com/boxboat/spire-tpm-plugin/pull/12 for the checksums on their releases to make it into their release notes.

Reply:
The plugins will have checksums on all releases now. see the text files here: https://github.com/boxboat/spire-tpm-plugin/releases on the newest releases.